src/api/contract/approve.js

```diff
@@ -21,7 +22,7 @@ const validateDepositAmountAndConvertToBN = amount => {
 
 const tokensWhichNeedResetToZero = ['USDT', 'OMG']
 
-module.exports = async (dvf, token, deposit, spender = dvf.config.DVF.starkExContractAddress) => {
+module.exports = async (dvf, token, deposit, chain = 'ETHEREUM', spender = dvf.config.DVF.starkExContractAddress) => {
```

Comment:
could move `chain` to the last argument to avoid breaking api change. If that will make the calls less pleasant, do we need to bump major and notify users of breaking changes?